bigtable/internal/transport/connpool.go

```diff
@@ -384,9 +385,37 @@ func NewBigtableChannelPool(ctx context.Context, connPoolSize int, strategy btop
 		btopt.Debugf(pool.logger, "bigtable_connpool: failed to create metrics reporter: %v\n", err)
 	}
 	pool.startMonitors()
+
+	// record the client startup time
+	// TODO: currently Prime() is non-blocking, we will make Prime() blocking and infer the transport type here.
+	transportType := "unknown"
+	pool.recordClientStartUp(startupTimer, transportType)
+
 	return pool, nil
 }
 
+func (p *BigtableChannelPool) recordClientStartUp(elapsedTimer time.Time, transportType string) {
+	if p.meterProvider != nil {
```

Comment:
nit: prefer
```go
if p.meterProvider == nil {
  return
}

...
clientStartupTime, err := meter.Float64Histogram(...
if err != nil {
  return
}

elapsed := float64(time.Since(elapsedTimer).Milliseconds())
```
This decreases levels of indentation and makes it easier to grasp normal code path.

go/go-style/decisions#indent-error-flow